engine/src/main/java/org/terasology/config/RenderingConfig.java

```diff
@@ -328,7 +328,7 @@ public void setVignette(boolean vignette) {
     }
 
     public boolean isMotionBlur() {
-        return motionBlur && !isOculusVrSupport();
+        return motionBlur && !isVrSupport();
```

Comment:
@indianajohn : this is obviously a legacy issue, but did you have a look at why this is? Why is motion blur incompatible with VR? This leaves us vulnerable to code that just relies on the property change event alone rather than, say, calling isMotionBlur or keeping track of both motion blur AND vr support property.